src/standalone/InlineSwitch/index.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useCallback } from "react";
+import { Grid, Typography, withStyles, Theme, Switch } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+export interface InlineSwitch {
+	/**
+	 * Value for switch position
+	 */
+	switchValue: boolean;
+	/**
+	 * Set value for switch position
+	 */
+	setSwitchValue: (checked: boolean) => void;
+	/**
+	 * Display switch control?
+	 */
+	displaySwitch: boolean;
+	/**
+	 * Label for switch control (only used if displaySwitch is truthy)
+	 */
+	switchLabel?: React.ReactNode;
+	children?: React.ReactElement;
+	/**
+	 * Apply custom margin top class to set UI
+	 */
+	customMarginTop?: boolean;
```

Comment:
instead of putting custom styles in this control you should implement style overrides. for an example look at Loader component.

Reply:
@TQjona , Changes updated. Please Review.